frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/obcPage.ts

```diff
@@ -0,0 +1,89 @@
+import { MINUTE } from '../utils/consts';
+import { projectNameSpace } from '../../../dev-console/integration-tests/support/pages/app';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+export class CreateOBCHandler {
+  name: string;
+
+  namespace: string;
+
+  storageclass: string;
+
+  bucketclass: string;
+
+  constructor(name: string, namespace: string, storageclass: string, bucketclass: string) {
```

Comment:
```suggestion
  constructor(name: string, namespace: string, storageclass: string) {
```
Since we are not using it anywhere.